server/container/metrics/metrics-es-reporter-v7/src/test/java/org/apache/james/metric/es/v7/ES7ReporterTest.java

```diff
@@ -20,9 +20,11 @@
 package org.apache.james.metric.es.v7;
 
 import org.apache.james.backends.es.v7.DockerElasticSearch;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-class ES7ReporterTest extends ESReporterContract {
+@Disabled
+class ES7ReporterTest { //TODO extends ESReporterContract {
```

Comment:
Why moving this extends as a TODO that is so easy to forget?

Reply:
Removed in https://github.com/apache/james-project/pull/288/commits/81ff164a5b6ee7afdbb2c1dfdfbd340e4cb88c38